.ci/release.groovy

```diff
@@ -68,9 +68,27 @@ pipeline {
         }
       }
     }
+    post {
+      post {
+        success {
+          slackMessage(statusMessage: "${PACKAGE_REGISTRY_DEPLOYMENT_NAME} Deployed success", color: 'good')
+        }
+        failure {
+          slackMessage(statusMessage: "${PACKAGE_REGISTRY_DEPLOYMENT_NAME} Deployed failure", color: 'warning')
+        }
+      }
+    }
   }
 }
 
+def slackMessage(Map args = [:]) {
+  def statusMessage = args.statusMessage
+  def color = args.color
+  slackSend(channel: '#integrations', color: "${color}",
```

Comment:
Do you think that Integrations is the right channel? Deployments may spam the channel, it's not like one deployment per day. How about a dedicated channel for such notifications or notify only about failed deployments?

Reply:
I like the idea of notifying only about failed deployments. But I'm also okay with starting with this PR as-is and if it gets too noisy, iterating on it to only report about failed deployments.

I'm -1 to the separate channel idea because IME it becomes very easy to ignore/mute a separate channel.

Reply:
I checked the Release Distribution activity and we can try with the Integrations channel. Let's stick with the Integrations, in the worst case we'll have motivation to adjust it :)

Reply:
It is easy to change if we see it is noisy